start.py

```diff
@@ -19,7 +19,6 @@
     handler.setFormatter(formatter)
     logger.addHandler(handler)
 
-    QCoreApplication.setAttribute(Qt.AA_EnableHighDpiScaling)
```

Comment:
May I know the reason for removing this line? IIRC I need to have this line to run termqt properly on my MacBook since it uses a high DPI screen.

Reply:
In qt6 the high DPI is on by default and cannot be turned off. However I putted back this functionality for qt5. See https://doc.qt.io/qtforpython-6/gettingstarted/porting_from2.html for details:

```
The High DPI scaling attributes Qt.AA_EnableHighDpiScaling, Qt.AA_DisableHighDpiScaling and Qt.AA_UseHighDpiPixmaps are deprecated. High DPI is by default enabled in Qt 6 and cannot be turned off.
```